<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gen_dialog</class>
 <widget class="QDialog" name="gen_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Batch generate images</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>File list</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="clear_file">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/icons/icons/delete.xpm</normaloff>:/icons/icons/delete.xpm</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="open_file">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/icons/icons/open.xpm</normaloff>:/icons/icons/open.xpm</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="file_list"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="type">
          <item>
           <property name="text">
            <string>Stain density imaging</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Stain size imaging</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Image Output</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="format">
          <item>
           <property name="text">
            <string>JPG file</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MAT file</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="tif_widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>min/max value</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="value_min"/>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="value_max">
             <property name="maximum">
              <double>100.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>5.000000000000000</double>
             </property>
             <property name="value">
              <double>5.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>resolution</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="resolution">
             <property name="minimum">
              <number>5</number>
             </property>
             <property name="maximum">
              <number>200</number>
             </property>
             <property name="singleStep">
              <number>5</number>
             </property>
             <property name="value">
              <number>40</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>microns</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Size min/max size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="min_size">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>50</number>
          </property>
          <property name="singleStep">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="max_size">
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="singleStep">
           <number>5</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>um</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="contour">
        <property name="text">
         <string>Add contour</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>gen_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>gen_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
